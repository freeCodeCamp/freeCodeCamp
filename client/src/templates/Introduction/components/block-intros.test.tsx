import React from 'react';
import renderer from 'react-test-renderer';
import BlockIntros from './block-intros';

const arrString = ['first paragraph', 'second paragraph'];

describe('<BlockIntros />', () => {
  it('matches snapshot', () => {
    const tree = renderer.create(<BlockIntros intros={arrString} />);
    expect(tree).toMatchSnapshot();
  });
});
