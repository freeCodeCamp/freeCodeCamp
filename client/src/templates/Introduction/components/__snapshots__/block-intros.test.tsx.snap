// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlockIntros /> matches snapshot 1`] = `
<div
  className="block-description"
>
  <p
    dangerouslySetInnerHTML={
      {
        "__html": "first paragraph",
      }
    }
  />
  <p
    dangerouslySetInnerHTML={
      {
        "__html": "second paragraph",
      }
    }
  />
</div>
`;
